soc/arm/microchip_mec/mec172x/reg/mec172x_gpio.h

```diff
@@ -20,12 +20,21 @@
 #define MCHP_GPIO_CTRL2_OFS	0x0500u
 
 /* MEC172XH-B0-SZ (144-pin) */
+#if defined(CONFIG_SOC_MEC172X_NSZ)
 #define MCHP_GPIO_PORT_A_BITMAP 0x7FFFFF9Du /* GPIO_0000 - 0036  GIRQ11 */
 #define MCHP_GPIO_PORT_B_BITMAP 0x7FFFFFFDu /* GPIO_0040 - 0076  GIRQ10 */
 #define MCHP_GPIO_PORT_C_BITMAP 0x07FFFCF7u /* GPIO_0100 - 0136  GIRQ09 */
 #define MCHP_GPIO_PORT_D_BITMAP 0x272EFFFFu /* GPIO_0140 - 0176  GIRQ08 */
 #define MCHP_GPIO_PORT_E_BITMAP 0x00DE00FFu /* GPIO_0200 - 0236  GIRQ12 */
 #define MCHP_GPIO_PORT_F_BITMAP 0x0000397Fu /* GPIO_0240 - 0276  GIRQ26 */
+#else
+#define MCHP_GPIO_PORT_A_BITMAP 0x7FFFFFFFu /* GPIO_0000 - 0036  GIRQ11 */
+#define MCHP_GPIO_PORT_B_BITMAP 0x7FFFFFFFu /* GPIO_0040 - 0076  GIRQ10 */
+#define MCHP_GPIO_PORT_C_BITMAP 0x3FFFFFF7u /* GPIO_0100 - 0136  GIRQ09 */
+#define MCHP_GPIO_PORT_D_BITMAP 0x3F67FFFFu /* GPIO_0140 - 0176  GIRQ08 */
+#define MCHP_GPIO_PORT_E_BITMAP 0x7FFFFFFFu /* GPIO_0200 - 0236  GIRQ12 */
+#define MCHP_GPIO_PORT_F_BITMAP 0x00003F7Fu /* GPIO_0240 - 0276  GIRQ26 */
```

Comment:
Shall we set GPIO251 and GPIO252 bits as 0? since it is not there in LJ package

Reply:
there's probably some clarification needed from Microchip on some things.   For example, GPIO253 exists on the NLJ package on pin E9 (according to the spec i have), but it doesn't appear to be able to cause a GIRQ.   Is this correct?  Or is this a miss in the datasheet.

Reply:
Yes, you are correct. Entry for GPIO253 missed out in the spec.
I raised the clarification to our System Engineering team.
Will update you the response.

Reply:
Its a spec update. Please find below System Engineering team response. They created ticket to update the spec.
![image](https://github.com/zephyrproject-rtos/zephyr/assets/123922411/4fa4823f-7fea-4eb8-ad6a-90aea9fc5377)

Reply:
Thanks for the quick response!  I'll update the patch and resubmit.